smelli/classes.py

```diff
@@ -197,8 +213,15 @@ def _load_likelihoods(self,
                 continue
             with open(self._get_yaml_path(fn), 'r') as f:
                 yaml_dict = flavio.io.yaml.load_include(f)
-                if not self.fix_ckm:
-                    yaml_dict['par_obj'] = par_ckm_dict
+                if fn in add_measurements.keys():
+                    yaml_dict['include_measurements'] += add_measurements[fn]
+                if fn in remove_measurements.keys():
+                    meas_missing = set(remove_measurements[fn])-set(yaml_dict['include_measurements'])
+                    if meas_missing:
+                        to_upgrade = 'smelli' if _flavio_up_to_date else 'flavio'
+                        raise AssertionError('The measurements {} have not been found. Please upgrade {} to the latest version.'.format(meas_missing,to_upgrade))
```

Comment:
I think in this case, the problem is not that `smelli` or `flavio` are not up to date such that one has to upgrade, but that one tries to remove a measurement that was not there in the first place. So I think we don't need the line defining `to_upgrade` and we can replace the error message by e.g.
```python
raise ValueErro(f'The measurements {meas_missing} are not part of {fn} and therefore cannot be removed.')
```
(I think it should be a `ValueError` in this case since the problem would be a wrong input)

Reply:
Yes, I had doubts between an `AssertError` or a `ValueError`. Your exception message sounds good to me.